deploy/test/main.tf

```diff
@@ -222,6 +251,47 @@ resource "aws_ecs_service" "git" {
 
 }
 
+resource "aws_ecs_service" "github" {
+  name            = "insights-github"
+  cluster         = aws_ecs_cluster.insights-git-cluster.id
```

Comment:
`s/aws_ecs_cluster.insights-git-cluster.id/aws_ecs_cluster.insights-github-cluster.id/`

Reply:
renamed the cluster to a common for all name.